project.clj

```diff
@@ -7,6 +7,13 @@
   :jvm-opts ["-Xss50M"]
   :dependencies [[org.clojure/clojure "1.9.0"]
                  [kixi/stats "0.4.0"]
-                 [criterium "0.4.4"]]
+                 [criterium "0.4.4"]
+                 [org.apache.commons/commons-math3 "3.6.1"]
```

Comment:
I don't see any use of math3, numeric-tower, or combinatorics in this PR, so I would be inclined to say remove them. If they serve some purpose, that purpose should be either documented or exploited.